### PR TITLE
security: changed sha1 calls to sha256

### PR DESCRIPTION
**Pull request by pacificcode**
Fixes [AB#580903](https://thycotic.visualstudio.com/4a89362e-1361-424f-a291-a8f57c2a8991/_workitems/edit/580903)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -64,6 +64,7 @@ docs/mkdocsenv/
 docs/site/
 dsv
 dsv.exe
+.dccache
```
Comment (sheldonhull) [inline .gitignore:67]: Curious what this is

**Review by sheldonhull — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (pacificcode) [reply .gitignore:67]: Best example I can find. https://github.com/snyk-labs/snyk-cicd-integration-examples/blob/master/.dccache

**Comment by pacificcode** [comment]
@sheldonhull can you check the status of this PR (seems to be stuck)

**Review by sheldonhull — APPROVED** [review]

**Comment by sheldonhull** [comment]
closing this due to coming from fork, and letting it be recreated as org contribution so all the integration tests run correctly.

**Pull request closed without merging**